hw_01/part_2/src/css/style.css

```diff
@@ -0,0 +1,486 @@
+/*-------------HEADER------------------------------------------------*/
+
+.header {
+  position: absolute;
+  width: 100%;
+  left: 0;
+  top: 0;
+  z-index: 50;
+  background-color: white;
+}
+
+.header .container {
+  display: flex;
+  min-height: 80px;
+  align-items: center;
+}
+@media screen and (max-width: 575px) {
+  .header .container {
+    flex-direction: column;
+    margin-top: 15px;
+    min-height: 160px;
+  }
+}
+
+.logo {
+  flex: 1 1 0;
+}
+
+@media screen and (max-width: 820px) {
+  .logo img {
+    width: 180px;
+  }
+}
+
+.menu {
+  margin: 0 3.82% 0 5%;
+}
+
+.menu .menu-list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-wrap: wrap;
+}
+@media screen and (max-width: 741px) {
+  .menu .menu-list {
+    justify-content: first baseline;
+  }
+}
+
+@media screen and (max-width: 575px) {
+  .menu .menu-list {
+    margin-top: 10px;
+    flex-direction: column;
+  }
+}
+
+.menu .item:not(:last-child) {
+  margin: 0 21px 0 0;
+}
+@media screen and (max-width: 820px) {
+  .menu .item:not(:last-child) {
+    margin: 0 12px 0 0;
+  }
+}
+
+@media screen and (max-width: 741px) {
+  .menu .item:not(:last-child) {
+    margin: 0 0 0 12px;
+  }
+}
+@media screen and (max-width: 575px) {
+  .menu .item:not(:last-child) {
+    margin: 0;
+  }
+}
+
+.menu .link {
+  font-size: 14px;
+  line-height: 129%;
+  letter-spacing: 0.03rem;
+  color: #000c2d;
+}
+@media screen and (max-width: 575px) {
+  .menu .link {
+    font-size: 16px;
+    line-height: 150%;
+  }
+}
+
+.menu .item.demo {
+  padding: 12px 23px;
+  border: 1px solid #2f70fe;
+}
+
+.menu .link.demo {
+  color: #2f70fe;
+}
+
+@media screen and (max-width: 575px) {
+  img.login-icon {
+    margin-top: 3px;
+  }
+}
+
+/*-------------PAGE------------------------------------------------*/
+.page {
+  position: relative;
+  margin-top: 80px;
+}
+@media screen and (max-width: 575px) {
+  .page {
+    margin-top: 224px;
+  }
+}
+/*-------------MAIN-BLOCK-------------------------------------------*/
+
+.picsbackground {
+  position: absolute;
+  width: 100%;
+}
+@media screen and (max-width: 768px) {
+  .picsbackground {
+    display: none;
+  }
+}
+
+.bgimages {
+  display: flex;
+
+  margin: 0, auto;
+  justify-content: center;
+}
+
+.imgleft {
+  margin: 0 200px 0 15px;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.imgright {
+  margin: 0 15px 0 200px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.imgleft img {
+  max-width: 100%;
+
+  padding-top: 290px;
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.imgright img {
+  max-width: 100%;
+  padding-top: 100px;
+
+  mix-blend-mode: luminosity;
+  opacity: 0.27;
+}
+
+.main {
+  position: relative;
+  background-color: #00ff8514;
+}
+
+.mainblock {
+  background-image: url("/src/img/mainblock/bg_blue.png");
```

Comment:
The same problem with relative path for `images` (I write comment about it in `html` file but here the path relative to css file where you use this image)